internal/serve/serve.go

```diff
@@ -311,6 +311,9 @@ func handleHTTP(o ServeOptions) *chi.Mux {
 
 			r.With(middleware.AnyRoleMiddleware(authManager, data.GetAllRoles()...)).
 				Patch("/", profileHandler.PatchUserProfile)
+
+			r.With(middleware.AnyRoleMiddleware(authManager, data.GetAllRoles()...)).
+				Patch("/reset-password", profileHandler.PatchUserPassword)
```

Comment:
Plz, don't forget to add this new endpoint to Test_handleHTTP_authenticatedEndpoints:

https://github.com/stellar/stellar-disbursement-platform-backend/blob/1fe89478630c1c3f8fe2d4ed8c51227a010bbeef/internal/serve/serve_test.go#L255-L278